pkg/operator/ceph/file/filesystem.go

```diff
@@ -249,3 +271,18 @@ func downFilesystem(context *clusterd.Context, clusterName, filesystemName strin
 	logger.Infof("Downed filesystem %s", filesystemName)
 	return nil
 }
+
+// GenerateMetadataPoolName is a helper function that generates a new name for the given metadata pool and returns the renamed pool.
+func GenerateMetadataPoolName(fsName string, metadataPool *model.Pool) *model.Pool {
```

Comment:
Where are these 2 new functions exported?

Reply:
agreed, if just in this file it should be a lower-case (package) method

Reply:
Because they were being used by the functions `newFS` and `SetPoolSize`, but I have changed their names to lowercase as they are only being used `rook/rook/pkg/operator/ceph/file/filesystem.go`.
Thanks for the alert.